auv_sim/auv_sim_bringup/launch/inc/start_pool_world.launch.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="debug" default="false" />
+  <arg name="namespace" default="taluy" />
+  <arg name="gui" default="true" />
+  <arg name="world_name" value="$(find auv_sim_description)/worlds/pool.world" />
+
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" value="$(arg world_name)" />
+    <arg name="use_sim_time" value="true" />
+    <arg name="gui" value="$(arg gui)" />
+    <arg name="headless" value="false" />
+    <arg name="debug" value="$(arg debug)" />
+    <arg name="verbose" value="true" />
+  </include>
+
+</launch>
```

Comment:
Separated the launch files for each world to make it easy and clean to add world-specific nodes